.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,59 @@
+name: Bug report
+description: File a bug report
+title: '[BUG]'
+labels: bug
+
+body:
+  - type: markdown
+    id: description
+    attributes:
+      value: |
+          ## Before posting a bug report
+          Search existing GitHub issues to make sure the issue does not already exist:
+          https://github.com/xanaduai/pennylane/issues
+
+          For general technical details check out our documentation:
+          https://pennylane.readthedocs.io
+
+          # Issue description
+          Description of the issue - include code snippets and screenshots here if relevant.
```

Comment:
Shouldn't code snippets be in the Source code section? 

Maybe we should specify that we prefer source code that reproduces the problem, but will accept screenshots if code is unable to sufficiently describe the problem.